### ISSUE TITLE
Updating to hax 2.8

### ISSUE THREAD

**Issue by PlayfulMaggot09** [yellows8/oot3dhax#110]
Not sure if this is necessarily an issue, on a 11.2.0-35U Old 3ds, and I'm trying to update from 2.7 to 2.8, whenever I run the oothax installer I get the yellow screen so I'm not sure how to get the paslr bypass without getting past it in the first place?

**Comment by rocket0634** [comment]
You'll need to install oot3dhax from another method of launching homebrew launcher that already has *hax 2.8 installed, or it needs to be installed on a 3ds lower than 11.0, or you'll need to use power saves for a cart.

**Comment by PlayfulMaggot09** [comment]
I have powersaves, how can I use that?

**Comment by rocket0634** [comment]
I don't know, I don't have a powersaves. I think you'll have to wait for someone to release the raw save images.

**Comment by yellows8** [comment]
" think you'll have to wait for someone to release the raw save images." Not doing that for a while.

**Comment by rocket0634** [comment]
@yellows8 is the sploit installer the only method to install oot3dhax 2.8 at the moment?

**Comment by PlayfulMaggot09** [comment]
Yeah I'ma need somebody to release the raw save images soon, idk how complicated it is or not but I'm gonna be patient

**Comment by yellows8** [comment]
"@yellows8 is the sploit installer the only method to install oot3dhax 2.8 at the moment?" <- yes

**Comment by HoLLy-HaCKeR** [comment]
@yellows8 So those that don't have another 2.8 entrypoint just need to wait for you to update the raw savegame which you "won't be doing for a while"?

I am using OoT3Dhax 2.7 right now, installed using a PowerSaves, and can't install any other *hax because of PASLR. Can't update to OoT3Dhax 2.8 through sploit_installer because of this.

**Comment by yellows8** [comment]
Indeed.

**Comment by HoLLy-HaCKeR** [comment]
I'm assuming there's more to it than just replacing the ROP payload then, otherwise that'd be a pretty dick move.

**Comment by PlayfulMaggot09** [comment]
I actually managed to get it through basehaxx!

**Comment by HoLLy-HaCKeR** [comment]
@PlayfulMaggot09 Did you install basehaxx before updating to 11.2 then? Since 11.1 and above has PASLR, it shouldn't be possible on 11.1 and 11.2 (it isn't for me, atleast).

**Comment by iSharingan** [comment]
JKSM has a PASLR bypass version (3dsx) that hijacks Pokebank (thus only works for OR/AS/Sun/Moon on hax 2.7). I might have to get that set up myself...

**Comment by HoLLy-HaCKeR** [comment]
Looks like you can install steelhax from 2.7, so that should work for most people.

**Comment by Korados** [comment]
So there won't be new save files in the near future? I had the same error as the OP. Unfortunately I can't run any of the other exploits because either I don't have the games or it's a game I'd like to keep the save file of. I hope that the saves get updated soon.

**Comment by grifnelle** [comment]
So let me get this straight, I need to launch homebrew on Oot with version 2.7 in order to install the basehax 2.8 to my system in order to install oothax 2.8 in order to install steelhax to 11.2

**Comment by yellows8** [comment]
Or wait for soundhax.

**Comment by grifnelle** [comment]
The process I stated (if possible) isn't gonna take that long, I'll just go and do it And when sound hax comes out be able to do Cia games on 11.2

**Comment by yellows8** [comment]
"Is soundhax ..." It's not done yet....

**Comment by iSharingan** [comment]
just an update: soundhax currently works for a few regions (I'm personally using the USA n3DS one with no problems). It requires an otherapp.bin and the appropriate soundhax.m4a for your 3DS (you shouldn't need to build the files yourself. Just use the ones on nedwill's Github). It can also be added purely by adding the two alongside the 2.8 *hax payload on your SD card.
